relation_engine_server/README.md

```diff
@@ -264,12 +264,14 @@ _Query params_
 
 Every call to update specs will reset the spec data (do a clean download and overwrite).
 
-### GET /api/v1/specs/schemas
+### GET /api/v1/specs/collections or /api/v1/specs/schemas
```

Comment:
Let's mark `/api/v1/specs/schemas` as deprecated in the docs.

Reply:
I think we can remove that path from the above header and just put a deprecation notice in this section somewhere (eg towards the bottom)